### DISCUSSION TITLE
Errore in configurazione automatica https

### DISCUSSION THREAD

**Discussion by proadelio** [globaleaks/globaleaks-whistleblowing-software#3821] [General]
Buonasera, nel pannello rete, tramite "Configurazione automatica" provo ad abilitare https. Ho impostato nel DNS l'url "whistleblowing.tesys-spa.it" (IP 2.33.12.229). Quando clicco su "Procedi", ricevo l'errore "Errore! Errore interno del server". Nel log /var/globaleaks/log/globaleaks.log trovo la seguente riga:
[E] exception mail suppressed for exception (<class 'acme.errors.ValidationError'>) [reason: special exception].
Qualcuno ha suggerimenti?

**Comment by andreasoc** [comment]
stesso identico problema, sto creando per la prima volta il certificato.
A livello dns ecc tutto perfetto

**Comment by andreasoc** [comment]
qualche idea ragazzi? Sarebbe utile in caso di queste problematiche, avere a disposizione il comando per generare il tutto manualmente compresa poi l'installazione.

**Comment by andreasoc** [comment]
Soluzione trovata grazie a @evilaliv3 
nella configurazione del NAT si deve tenere aperta anche la porta 80 che serve a Let's encrypt per generare il certificato!
grazie di cuore
